README.md

```diff
@@ -88,8 +84,14 @@ if (measurements.cpu)
     const auto &counter = measurements.cpu->find(CpuCounter::Cycles);
```

Comment:
this is not correct, since measurements is now an std::pair, so this part of documentation has to be updated